src/ray/core_worker/lib/java/org_ray_runtime_task_NativeTaskSubmitter.cc

```diff
@@ -141,7 +146,7 @@ JNIEXPORT jobject JNICALL Java_org_ray_runtime_task_NativeTaskSubmitter_nativeSu
   std::vector<ObjectID> return_ids;
   auto status = GetCoreWorker(nativeCoreWorkerPointer)
                     .SubmitTask(ray_function, task_args, task_options, &return_ids,
-                                /*max_retries=*/1);
+                                /*max_retries=*/0);
```

Comment:
what's this for?

Reply:
The default value of `max_retries` is `0`. Without the fix, it will break `testGetThrowsQuicklyWhenFoundException`. The test case expects that we can get the exceptional return object of the failed task quickly, while `max_retries == 1` means that failed tasks will be retried once with a delay of 5s before we can get the return object.

Reply:
Could you add a TODO to allow user to specify `max_retries` per call? Currently it's harded for java here.

Reply:
Done.